mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/CheckCmd.scala

```diff
@@ -51,13 +48,13 @@ class CheckCmd(name: String = "check", description: String = "Check a TLA+ speci
 
   var maxError: Option[Int] =
     opt[Option[Int]](name = "max-error",
-        description =
-          "do not stop on first error, but produce up to a given number of counterexamples (fine tune with --view), default: 1",
-        default = None)
+      description =
+        "do not stop on first error, but produce at most the given number of errors (requires --view when greater than 1), default: 1",
+      default = None)
 
   var view: Option[String] =
-    opt[Option[String]](name = "view",
-        description = "the state view to use with --max-error=n, default: transition index", default = None)
+    opt[Option[String]](name = "view", description = "the state view to use with --max-error=n, default: none",
+      default = None)
```

Comment:
iiuc, the logic around this indicates that `view` is only useful when `max-error` is used and `max-error` is only useful when `view` is used. Doesn't this indicate that these should be combined into a single flag?

Could it be that both of these flags are really named after parts of the implication details, and what we actually want is some sort of flag like: `--view-error-traces=view:n`?

Just a thought. Would be curious to know what @thpani thought on this point.

Reply:
@konnov do we still plan to have better default behavior when `--view` is not provided?
Otherwise I agree with Shon, we should either put them into a single flag or at least have two flags with similar names like `--max-error.num=N` and `--max-error.view=View`.

Reply:
There is #2143, which should work without view.